### ISSUE TITLE
Mouse Wheel Not always responding in CameraUI

### ISSUE THREAD

**Issue by bradrothenberg** [cinder/Cinder#881]
Hello,

The mouseWheel() in CameraUI seems to be thinking it should exit the function early -- I am using windows 7, & the latest GLNext branch, with the new camera API.

A normal mouse wheel scroll is still exiting @ the following if-statement:

```c++
	// some mice issue mouseWheel events during middle-clicks; filter that out
	if( mLastAction != ACTION_NONE )
		return;
```

**Comment by andrewfb** [comment]
Hi - can you describe how to reproduce the case you're seeing more specifically, say, against the FrustumCullingRedux sample that uses it?

**Comment by bradrothenberg** [comment]
Hi Andrew,

I can reproduce the case by initializing the camera without connecting it to the window:
```c++
mCamUi = CameraUi( &mRenderCam);
```

 & passing the MouseEvent's through to the CameraUI as follows:
```c++
void FrustumCullingReduxApp::mouseDown(MouseEvent event)
{
	mCamUi.mouseDown(event);

}

void FrustumCullingReduxApp::mouseDrag(MouseEvent event)
{
	mCamUi.mouseDrag(event.getPos(), (event.isRightDown() && !event.isShiftDown()), (event.isRightDown() && event.isShiftDown()), (event.isRightDown() && event.isControlDown()));

}

void FrustumCullingReduxApp::mouseWheel(app::MouseEvent event)
{
	mCamUi.mouseWheel(event.getWheelIncrement());
}
```

After starting the app, the mouse-wheel input only works after left-clicking, If I right-click to rotate, the mouse-wheel callback takes me to this line (line 226 of CameraUI.cpp), where it returns: 

```c++
 	if( mLastAction != ACTION_NONE )
 		return;
```

**Comment by andrewfb** [comment]
I think this is actually 2 separate issues; the first should be addressed by 2adf685. The second is that CameraUi also needs mouseUp events so that it knows when a drag is complete. That 'mLastAction' logic is in there because some mice send mouseWheel events during a middle drag. Let us know if these changes fix the behavior in your own application...

**Comment by bradrothenberg** [comment]
Yup, that fixed it.

Thanks!
